src/commands.rs

```diff
@@ -217,14 +218,15 @@ mod poll {
             log::debug!("Removing quote message");
             bot.delete_message(dialogue.chat_id(), msg.id).await?;
 
-            let split_pos = (config().committee.len() / 2) as u8;
+            let split_pos = 10 as u8; // max poll options
 
-            // Splits the committee and add an option to refer to the other poll.
-            let polls = config().committee.split_at(split_pos as usize);
-            let polls = (
-                [polls.0, &["J'ai voté en dessous".to_owned()]].concat(),
-                [polls.1, &["J'ai voté en dessus".to_owned()]].concat(),
-            );
+            // Splits the committee to have only 10 answers possible.
+            let mut poll = config().committee.clone();
+            poll.retain(|s| -> bool {*s != target});    // filter the target from options
+            poll.shuffle(&mut thread_rng());                // shuffle the options
+            let index = thread_rng().gen_range(0..split_pos); // generate a valid index to insert target back
+            poll.insert(index as usize, target.clone());        // insert target back in options
```

Comment:
The resulting `Vec` will have a size of 11.
```suggestion
            let index = thread_rng().gen_range(0..(split_pos-1)); // generate a valid index to insert target back
            poll.insert(index as usize, target.clone());        // insert target back in options
```